cmd/cmd.go

```diff
@@ -280,12 +280,90 @@ func createBlob(cmd *cobra.Command, client *api.Client, path string) (string, er
 	}
 
 	digest := fmt.Sprintf("sha256:%x", hash.Sum(nil))
+
+	// Here, we want to check if the server is local
+	// If true, call, createBlobLocal
+	// This should find the model directory, copy blob over, and return the digest
+	// If this fails, just upload it
+	// If this is successful, return the digest
+
+	// Resolve server to IP
+	// Check if server is local
+	if client.IsLocal() {
```

Comment:
Note: this check may not work if Ollama is running in Docker, which often exposes it on localhost:11434